phylogenetic/rules/prepare_sequences.smk

```diff
@@ -83,9 +83,9 @@ rule filter:
         "logs/{group}/{gene}/filter.txt",
     params:
         id_field = config['strain_id_field'],
-        min_length = config['filter']['min_length'],
```

Comment:
Looks like `min_length` was already not used in the filter...should remove from config.yaml to reduce confusion. Additional nit to update the docstring for this rule to remove teh params that are no longer used.

Reply:
Ah, thanks! Dropped as part of https://github.com/nextstrain/norovirus/pull/21/commits/098b35f5bc3a608846f44259cc73ea750323422d and https://github.com/nextstrain/norovirus/pull/21/commits/7c0098526b6d9a130ae75230a4118a9c4215e914